recipes/entrez-direct/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: entrez-direct
+  version: "3.90"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: edirect.tar.gz
+  url: ftp://ftp.ncbi.nlm.nih.gov/entrez/entrezdirect/versions/2016-03-23/edirect.tar.gz
+  sha256: c88f929b1b0b94c341746b3b50c9f4e3fbd7cb0d65f6d8565feb8d1016cad8a3
+
+requirements:
+  build:
+  - libgcc
```

Comment:
`libgcc` is not a build requirement, isn't it? Do you mean gcc? If so can you include gcc as linux (# [linux]) and llvm (# [osx]) requirement?